configfs.c

```diff
@@ -115,6 +136,18 @@ char *tcmu_get_wwn(struct tcmu_device *dev)
 	return ret_buf;
 }
 
+int tcmu_set_device_size(struct tcmu_device *dev, bool with_nl)
```

Comment:
Rename to tcmu_set_dev_size to match the other naming.

I think we can also drop the with_nl argument, because we probably never want to call into ourself and outside users doing the other model that does not use rtslib also do not want nl notifications into themself.

Reply:
Yes, sure, will fix it.